01_run_asli_calculations.sh

```diff
@@ -9,6 +9,6 @@ echo "Requesting with the following arguments: $DATA_ARGS_ERA5".
 asli_data_era5 $DATA_ARGS_ERA5
 
 # Run calculation, specifying output location
-# output.csv will need to be renamed to sensible unique identifyer 
-asli_calc $DATA_DIR/era5_mean_sea_level_pressure_monthly_*.nc -o $OUTPUT_DIR/asli_calculation_$FILE_IDENTIFYER.csv
+# output.csv will need to be renamed to sensible unique IDENTIFIER 
```

Comment:
Are the capitals intentional here?

Reply:
A classic case of ctrl + F replace, good catch